pkg/apis/pingcap/v1alpha1/validation/validation.go

```diff
@@ -643,6 +644,21 @@ func validateUpdatePDConfig(oldPdSpec, pdSpec *v1alpha1.PDSpec, path *field.Path
 	return allErrs
 }
 
+func disallowMutateBootstrapSQLConfigMapName(old, new *v1alpha1.TiDBSpec, p *field.Path) field.ErrorList {
```

Comment:
it seems there are no need to restrict this

Reply:
Because it only take effect at the first bootstrap, so updating the field makes no sense but make tidb restart.